File: 프로그래머스/lv0/120883. 로그인 성공？/로그인 성공？.py
def solution(id_pw, db):
    for i in db:
        print(i)
        if id_pw == i:
            answer = "login"    
        elif id_pw[0] == i[0] and id_pw[1] != i[1]:
            answer = "wrong pw"
        elif id_pw[0] != i[0] and id_pw[1] != i[1]:
            answer = "fail"
    return answer

File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.java
import java.util.Arrays;
import java.util.OptionalInt;

class Solution {
    public String solution(String[] idPw, String[][] db) {
        String answer = "fail";
        for (String[] row : db){
            if (idPw[0].equals(row[0])){
                if (idPw[1].equals(row[1])){
                    answer = "login";
                    break;
                }
                answer = "wrong pw";
            }
        }
        return answer;
    }

    public static void main(String[] args) {
        Solution T = new Solution();
        String[] idPw = {"meosseugi", "1234"};
        String[][] db = {{"rardss", "123"}, {"yyoom", "1234"}, {"meosseugi", "1234"}};


        System.out.println(T.solution(idPw, db));
    }
}
